### PR TITLE
Add descriptions to resource page

### PR DESCRIPTION
**Pull request by ZanderSparrow**
Replace the placeholder with actual info

### CODE REVIEW

File: app/components/Resource/Resource.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ class Resource extends Component {
                 </div>
               </div>
               <div className="org-desc">
-                <p>This is an organisation's long description{this.props.description}</p>
+                <p>{this.state.resource.long_description || this.state.resource.short_description || "Description of the resource"}</p>
```
Comment (jfhamlin) [inline app/components/Resource/Resource.js]: Do we really want "Description of the resource" as a fallback?

Comment (ZanderSparrow) [inline app/components/Resource/Resource.js]: "No description available"? or just blank?

Comment (jfhamlin) [inline app/components/Resource/Resource.js]: I'm curious what @derekfidler thinks.

Comment (achadha) [inline app/components/Resource/Resource.js]: Pinged Derek on Slack, but if he doesn't respond by tomorrow, let's just change it to "No description available." Other than that, this looks good to merge.

Comment (derekfidler) [inline app/components/Resource/Resource.js]: no description available is preferable.

**Pull request merged**